core/deployment/src/main/java/io/quarkus/deployment/builditem/InitTaskCompletedBuildItem.java

```diff
@@ -7,13 +7,13 @@
  * Similar to {@link ServiceStartBuildItem} but focused on initialization tasks (e.g. db migrations etc) that are run during
  * runtime just before the application startups.
  * <p>
- * The build item is used, so that we can track when all intialization tasks have been completed.
+ * The build item is used, so that we can track when all initialization tasks have been completed.
  */
-public final class InitalizationTaskCompletedBuildItem extends MultiBuildItem {
+public final class InitTaskCompletedBuildItem extends MultiBuildItem {
```

Comment:
I'm wondering if we should deprecate this for a version before deleting it.

@ppalaga are you using that in Camel Quarkus by any chance?

Reply:
> I'm also not sure if this build item is used outside of the core repository.

Given that it was introduced around November 2022, I'd say that it's not very likely. Sourcegraph search also seems to agree (though I am not sure if it can be trusted).

Reply:
@iocanel the class is present since 3.0.0.Alpha3 and it was introduced in Jan 16th in this commit:   https://github.com/quarkusio/quarkus/commit/c4d34d1e878bd847e297e77f374fa38401180a7d


So I'd say it's perfectly fine to remove it before it lands into a stable release.